src/Security/XpubTokenGenerator.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace eLife\Journal\Security;
+
+use Firebase\JWT\JWT;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+final class XpubTokenGenerator
+{
+    const TOKEN_TTL = 60;
+
+    private $secret;
+
+    public function __construct(string $secret)
+    {
+        $this->secret = $secret;
+    }
+
+    public function generate(UserInterface $user) : string
+    {
+        return JWT::encode([
+            'iat' => time(),
+            'exp' => time() + self::TOKEN_TTL,
+            'id' => $user->getUsername(),
```

Comment:
Would it be possible to add an `iss`uer claim so we can easily distinguish the Journal generated tokens from the xPub ones?

Reply:
Yep, we do that for Hypothesis: https://github.com/elifesciences/journal/blob/d389d957214073f7db36637882fbf7ed36684551/src/Security/HypothesisTokenGenerator.php#L25-L31

Reply:
I thought the `iss` was the issuer (i.e. 'journal' not the client). Am I wrong?

Reply:
That’s what I’d expect it to be (or `journal--prod` etc). Was keeping the same terminology, though maybe it’s wrong here?